### PR TITLE
[DO NOT MERGE] NO-ISSUE: Renamed SWF VS Code extension to Openshift Serverless Logic Editor and added update-version-to option to maven build

### PR DESCRIPTION
**Pull request by paulovmr**
**DO NOT MERGE as the scope of this PR is still being discussed and validated.**

This PR includes the renamed Openshift Serverless Logic Editor VS Code extension to the maven build.

To execute the maven build, it is optional to export a custom NPM registry URL for dependencies download, like:
`export NPM_REGISTRY_URL=https://registry.npmjs.org`

When doing a release build, it is necessary to pass the version of the NPM packages as a property. It has to follow the `x.y.z` pattern [[1]](https://docs.npmjs.com/about-semantic-versioning) [[2]](https://docs.npmjs.com/creating-a-package-json-file). I've added an automation for that in the maven build, so it can just be executed like this:
`mvn clean install -DpackagesVersion=1.33.0`

[1] [https://docs.npmjs.com/about-semantic-versioning](https://docs.npmjs.com/about-semantic-versioning)
[2] [https://docs.npmjs.com/creating-a-package-json-file](https://docs.npmjs.com/creating-a-package-json-file)

-----

**To do:**
* Change the extension logo (FYI @ricardozanini )

### CODE REVIEW

File: packages/serverless-workflow-vscode-extension/README.md [hunk 1]
```diff
@@ -1,7 +1,6 @@
-## Kogito Serverless Workflow Editor
+## Openshift Serverless Logic Editor
```
Comment (paulovmr) [inline packages/serverless-workflow-vscode-extension/README.md:1]: @ricardozanini @domhanak Let me know if you want any changes on the extension readme. This goes on the extension page in the marketplace.

File: packages/serverless-workflow-vscode-extension/package.json [hunk 2]
```diff
@@ -89,7 +89,7 @@
   "engines": {
     "vscode": "^1.67.0"
   },
-  "displayName": "KIE Serverless Workflow Editor",
+  "displayName": "Openshift Serverless Logic Editor",
```
Comment (caponetto) [inline packages/serverless-workflow-vscode-extension/package.json:92]:
```suggestion
  "displayName": "OpenShift Serverless Logic Editor",
```

File: packages/serverless-workflow-vscode-extension/README.md (hunk 1 above)
Comment (caponetto) [inline packages/serverless-workflow-vscode-extension/README.md:1]:
```suggestion
## OpenShift Serverless Logic Editor
```

File: packages/serverless-workflow-vscode-extension/CHANGELOG.md [hunk 3]
```diff
@@ -1,184 +1,3 @@
...
-- [KOGITO-6645](https://issues.redhat.com/browse/KOGITO-6645) - Serverless Workflow VS Code Extension
-- [KOGITO-6558](https://issues.redhat.com/browse/KOGITO-6558) - [SW Editor] - YAML support
-- [KOGITO-6646](https://issues.redhat.com/browse/KOGITO-6646) - Marshaller Issues
-- [KOGITO-6647](https://issues.redhat.com/browse/KOGITO-6647) - Upgrade Extension for 0.8 specification
-- [KOGITO-6648](https://issues.redhat.com/browse/KOGITO-6648) - Migrate Extension to Apache KIE Tools Repository
-- [KOGITO-6795](https://issues.redhat.com/browse/KOGITO-6795) - Serverless Workflow Editor Issues
-- [KOGITO-6807](https://issues.redhat.com/browse/KOGITO-6807) - Implement undo, redo and isDirty
-- [KOGITO-6796](https://issues.redhat.com/browse/KOGITO-6796) - Fetch error on console when opening the editor
-- [KOGITO-6828](https://issues.redhat.com/browse/KOGITO-6828) - Monaco font is not configured properly
+First version of Openshift Serverless Logic Editor, where you can create and edit Serverless Workflow definition files (\*.sw.json, \*.sw.yaml, \*.sw.yml).
```
Comment (caponetto) [inline packages/serverless-workflow-vscode-extension/CHANGELOG.md:3]:
```suggestion
First version of OpenShift Serverless Logic Editor, where you can create and edit Serverless Workflow definition files (\*.sw.json, \*.sw.yaml, \*.sw.yml).
```

**Pull request closed without merging**